omnia/scuttlebot.sh

```diff
@@ -76,13 +79,20 @@ broadcastPriceMsg () {
         error "Error - failed to transpose sources with prices"
         return
     fi
+    #format starkware sig
+
+    _starkSignature=( "--arg r $_starkSignatureR" "--arg s $_starkSignatureS" "--arg publicKey $_starkPublicKey" )
+    verbose  "${_starkSignature[*]}"
+    if ! _starkSignatureJson=$(jq -nce  --arg r "$_starkSignatureR" --arg s "$_starkSignatureS" --arg publicKey "$_starkPublicKey" '{r: $r, s:$s, publicKey:$publicKey}'); then
+        error "Error - failed to generate stark signature json"
+    fi
     #compose jq message arguments
-    _jqArgs=( "--arg assetPair $_assetPair" "--arg version $OMNIA_VERSION" "--arg price $_price" "--arg priceHex $_priceHex" "--arg time $_time" "--arg timeHex $_timeHex" "--arg hash ${_hash:2}" "--arg signature ${_signature:2}" "--argjson sourcePrices $_sourcePrices" )
+    _jqArgs=( "--arg assetPair $_assetPair" "--arg version $OMNIA_VERSION" "--arg price $_price" "--arg priceHex $_priceHex" "--arg time $_time" "--arg timeHex $_timeHex" "--arg hash ${_hash:2}" "--arg signature ${_signature:2}" "--argjson starkSignature $_starkSignatureJson" "--argjson sourcePrices $_sourcePrices" )
     #debug
     verbose "${_jqArgs[*]}"
     #generate JSON msg
     # shellcheck disable=2068
```

Comment:
Remove this as it warns about splitting parameter substitutions which should be fixed, see next comment.

```suggestion
```